<launch>

  <arg name="topic" default="ugr/car/map" />
  <arg name="filename" default="rc_gt1.yaml" />

  <node name="map_publisher" pkg="slam_simulator" output="screen" type="map_publisher.py">
    <param name="map" value="$(find slam_simulator)/maps/$(arg filename)" />

    <remap from="/output/map" to="$(arg topic)" />
  </node>

</launch>